webapp-django/crashstats/crashstats/views.py

```diff
@@ -1457,6 +1457,13 @@ def report_index(request, crash_id, default_context=None):
                         args=(crash_id, name, 'dmp')
                     )
                 )
+        if context['raw'].get('ContainsMemoryReport'):
```

Comment:
Please also check that the ``memory_report`` key exists, because at some point in the near future, I am going to change the behavior of the processors so they do not create that key if the memory report is invalid. The report would, in that case, not have the ``memory_report`` key but a ``memory_report_error`` key. See https://bugzilla.mozilla.org/show_bug.cgi?id=1266601 for more details.